content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/inventory_valuation_config.rst

```diff
@@ -9,78 +9,95 @@ be reflected in the company's accounting records to accurately show the value of
 all of its assets.
 
 By default, Odoo uses a periodic inventory valuation (also known as manual inventory valuation).
-This method implies that the accounting team posts journal entries based on the physical inventory
-of the company, and that warehouse employees take the time to count the stock. In Odoo, this method
-is reflected inside each product category, where the :guilabel:`Costing Method` field will be set to
-`Standard Price` by default, and the :guilabel:`Inventory Valuation` field will be set to `Manual`.
+This method implies that the accounting team manually posts journal entries based on the physical
+inventory of the company, and that warehouse employees take the time to count the stock. In Odoo,
+each product category reflects this with the :guilabel:`Costing Method` set to :guilabel:`Standard
+Price` and the :guilabel:`Inventory Valuation` (not visible by default) set to :guilabel:`Manual`.
 
 .. image:: inventory_valuation_config/inventory-valuation-fields.png
    :align: center
-   :alt: The Inventory Valuation fields are located on the Product Categories form.
+   :alt: The Costing Method field is located on the Product Categories form.
 
-Alternatively, automated inventory valuation is an integrated valuation method that updates the
-inventory value in real-time by creating journal entries whenever there are stock moves initiated
-between locations in a company's inventory.
+Alternatively, perpetual (automatic) inventory valuation creates real-time *journal entries* in the
+*Accounting* app whenever stock enters or leaves the company's warehouse.
 
-.. note::
-   Automated inventory valuation is a method recommended for expert accountants, given the extra
-   steps involved in journal entry configuration. Even after the initial setup, the method will
-   need to be periodically checked to ensure accuracy, and adjustments may be needed on an ongoing
-   basis depending on the needs and priorities of the business.
+This document is focused on the proper setup of automatic inventory valuation, which is an
+integrated valuation method that ensures journal entries in *Accounting* match stock valuation
+updates in *Inventory*. For an introduction of inventory valuation in Odoo, refer to the
+:doc:`using_inventory_valuation` documentation.
 
-.. _inventory/inventory_valuation_config/accounting:
+.. warning::
+   Switching to automatic inventory valuation **will not** create journal entries for existing
+   stock, potentially leading to discrepancies between stock valuation and accounting journals.
 
-Types of accounting
-===================
+   To manually post journal entries for existing stock, navigate to :menuselection:`Inventory app
+   --> Reporting --> Valuation` for a list of current stock and its value. Then, :doc:`post a
+   journal entry <../../../../finance/accounting/vendor_bills>` for each existing item.
```

Comment:
@toaa-odoo Is this the best accounting document to refer to for readers who are looking to manually post journal entries?

Reply:
Yeah it is fine